ministryofjustice/correspondence_tool_staff
# Cdpt 616 update security records template

## Description
Added 'Colleague' to the salutation of the security records template 

## Self-review checklist

* [ *] (3) Tests passing
* [ *] (4) Branch ready to be merged (not work in progress)
* [ *] (5) No superfluous changes in diff
* [ *] (6) No TODO's without new ticket numbers
* [ *] (7) PR Prefixed with ticket number e.g. `CT-7654 ...`


### Screenshots
<img width="763" alt="Screenshot 2023-04-13 at 15 42 35" src="https://user-images.githubusercontent.com/129938801/231796233-b53833ff-ec28-4c76-9ee3-04b9aaaa0875.png">


### Related JIRA tickets
https://dsdmoj.atlassian.net/jira/software/c/projects/CDPT/boards/1152?modal=detail&selectedIssue=CDPT-616

### Deployment
N/A

### Manual testing instructions
GIVEN: I have selected the ‘Security records’ Day 1 commissioning template
AND: Have selected ‘Download’
WHEN: I view the template
THEN: I see the new static content for the greeting Dear Colleague (see attached template for example)

GIVEN: I have selected the ‘Security records’ Day 1 commissioning template
AND: Have selected ‘Download’
WHEN: I view the template
THEN: I see the new dynamic content below the greeting, displaying the selected organisation name from the address book (see attached template for example).

GIVEN: I edit the data request, changing the 'Location`
AND: I have selected the ‘Security records’ Day 1 commissioning template
AND: Have selected ‘Download’
WHEN: I view the template
THEN: I see the new static content for the greeting Dear Colleague (see attached template for example)

GIVEN: I edit the data request, changing the 'Location`
AND: I have selected the ‘Security records’ Day 1 commissioning template
AND: Have selected ‘Download’
WHEN: I view the template
THEN: I see the new dynamic content below the greeting, displaying the selected organisation name from the address book (see attached template for example).
